spec/api-browser-window-spec.js

```diff
@@ -419,6 +419,14 @@ describe('BrowserWindow module', () => {
     })
   })
 
+  describe('BrowserWindow.getFocusedWindow()', () => {
+    it('returns the opener window when dev tools window is focused', () => {
+      w.webContents.openDevTools({mode: 'undocked'})
+      w.show()
```

Comment:
Does this ensure that the devtools window is focused? My intuition is that the `w.show()` here would mean the browserwindow would be focused.

Reply:
Done! Thanks! Changed the test a little.